sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -129,14 +129,14 @@ private[sql] class HiveSessionCatalog(
     Try(super.lookupFunction(funcName, children)) match {
       case Success(expr) => expr
       case Failure(error) =>
-        if (functionRegistry.functionExists(funcName)) {
-          // If the function actually exists in functionRegistry, it means that there is an
-          // error when we create the Expression using the given children.
+        if (super.functionExists(name)) {
```

Comment:
This should be replaced by `functionExists(name)`

Reply:
We should keep use `super.functionExists(name)`, we can not load  a Hive's built-in function  if replaced by `functionExists(name)` and `org.apache.spark.sql.AnalysisException: Undefined function: 'histogram_numeric'` will be thrown.